### PR TITLE
feat: add control to fill the city input

### PR DESCRIPTION
**Pull request by ardasnturk**
- added alert to notify that if the postcode input is filled in, the city input must also be filled in to prevent the city input from being empty
- added a new alert to fill in the street input if the house number is entered
- added * for required inputs

MQGB-60

## Screenshots:

|post code alert|city alert|
|--|--|
Simulator Screenshot - iPhone 15 Pro Max - 2024-02-26 at 13 03 04|Simulator Screenshot - iPhone 15 Pro Max - 2024-02-26 at 13 03 09

### CODE REVIEW

File: src/config/texts.js [hunk 1]
```diff
@@ -933,6 +933,7 @@ export const texts = {
       alerts: {
         address: 'Bitte stellen Sie sicher, dass Sie Ihre Adressdaten korrekt eingeben',
         contact: 'Bitte stellen Sie sicher, dass Sie Ihre Kontaktdaten korrekt eingeben.',
+        city: 'Bitte füllen Sie den Abschnitt Ort aus',
```
Comment (donni106) [inline src/config/texts.js:936]: Bitte geben Sie den Ort an

File: src/config/texts.js [hunk 2]
```diff
@@ -956,9 +957,11 @@ export const texts = {
         no: 'Nein',
         myLocation: 'Möchten Sie Ihren aktuellen Standort verwenden?',
         serviceCode: 'Bitte wählen Sie aus, um welches Thema es in dem Bericht geht.',
+        street: 'Bitte füllen Sie den Abschnitt Straße aus',
```
Comment (donni106) [inline src/config/texts.js:960]: Bitte geben Sie die Straße an

File: src/config/texts.js [hunk 3]
```diff
@@ -956,9 +957,11 @@ export const texts = {
...
         termsOfService: 'Bitte akzeptieren Sie die Datenschutzbestimmungen.',
         title: 'Bitte kurz beschreiben, worum es geht.',
         yes: 'Ja',
+        zipCode: 'Bitte füllen Sie den Abschnitt Postleitzahl aus',
```
Comment (donni106) [inline src/config/texts.js:964]: Bitte geben Sie die Postleitzahl an

File: src/screens/SUE/SueReportScreen.tsx [hunk 4]
```diff
@@ -211,9 +211,21 @@ export const SueReportScreen = ({
...
+          return texts.sue.report.alerts.zipCode;
+        }
+
+        if (getValues().zipCode.length !== 5) {
           return texts.sue.report.alerts.zipCodeLength;
         }
+
+        if (!getValues().city) {
+          return texts.sue.report.alerts.city;
+        }
```
Comment (donni106) [inline src/screens/SUE/SueReportScreen.tsx:228]: we should add `*` to these two fields, right?

Comment (ardasnturk) [reply src/screens/SUE/SueReportScreen.tsx:228]: Yes 👍
 https://github.com/smart-village-solutions/smart-village-app-app/commit/381cfc039e76f6793924224c2df3cbb1447a7aa1

**Review by donni106 — APPROVED** [review]

**Pull request merged**